flags/src/main/java/com/yahoo/vespa/flags/Flags.java

```diff
@@ -259,6 +259,12 @@
             "Takes effect at redeployment",
             ZONE_ID, APPLICATION_ID);
 
+    public static final UnboundDoubleFlag MAX_DEAD_BYTES_RATIO = defineDoubleFlag(
+            "max-dead-bytes-ratio", 0.2,
+            List.of("baldersheim", "geirst","toregge"), "2021-02-03", "2022-02-03",
```

Comment:
Let it expire at end of Q1 2021-04-01: We should have rolled out a sensible default value by then.